File: CmdMents/God/KnowGodExists.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CmdMents.God
{
    class KnowGodExists : LoveGodWithHeartSoulStrength
    {
        public KnowGodExists()
        {
            base.Book = CommandmentBook.Exodus;
            base.CanBeCarriedOutToday = true;
            base.Chapter = 20;
            base.CommandmentType = CommandmentType.Positive;
            base.FollowedByChristians = CommandmentObedience.Obeyed;
            base.FollowedByMessianics = CommandmentObedience.Obeyed;
            base.FollowedByObservantJews = CommandmentObedience.Obeyed;
            base.Number = 1;
            base.ShortSummary = "Know that God exists.";
            base.Text = "I am ADONAI your God, who brought you out of the land of Egypt, out of the abode of slavery.";
            base.Verse = 2;
        }
    }
}

File: EtzMitzvot/Commandments/God/knowGodExists.ts
///<reference path='../commandmentBase.ts' />

class KnowGodExists extends CommandmentBase {
    constructor() {
        super();

        this.shortSummary = "Know that God exists.";
        this.text = "I am the LORD your God, who brought you out of the land of Egypt, out of the abode of slavery.";
        this.book = TorahBook.Exodus;
        this.chapter = 20;
        this.verse = 2;

        this.canBeCarriedOutToday = true;
        this.requiresLivingInIsrael = false;

        this.type = CommandmentType.Positive;
        this.christianObservance = CommandmentObedience.Binding;
        this.messianicObservance = CommandmentObedience.Binding;
        this.jewishObservance = CommandmentObedience.Binding;
        this.number = 1;

        this.children = [
            new KnowGodIsOne(), new AcknowledgeGodAsHoly()
        ];
    }
}